physchem_calc.py

```diff
@@ -97,7 +108,9 @@ def read_smi(fname, sep="\t"):
                                                  'fmf: fraction of atoms belonging to Murcko framework\n'
                                                  'QED: quantitative estimate of drug-likeness\n'
                                                  'HAC: heavy atom count\n'
-                                                 'NumRingsFused: number of rings considering fused and spirocycles as a single ring',
+                                                 'StereoCenters: number of possible stereoisomers for a molecule\n'
+                                                 'NumRingsFused: number of rings considering fused and spirocycles as a single ring\n'
+                                                 'Num_HBA_HBD_atoms: number of H-bond acceptors and H-bond donors atoms\n',
```

Comment:
rephrase:
unique_HBAD: number of unique H-bond acceptors and H-bond donors atoms

Reply:
6